pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# References <!-- Link to the Jira task -->
```

Comment:
I'd keep this a bit more low key than a `h1`. 😅 

```suggestion
**References** <!-- Link to the Jira task -->
```

Reply:
Ha yes I would like this too.

Reply:
lol no idea how it ended up like that